### DISCUSSION TITLE
Test case attachement if flow fails at any stage during ORFS

### DISCUSSION THREAD

**Discussion by vijayank88** [The-OpenROAD-Project/OpenROAD-flow-scripts#377] [Q&A] [answered]
Hi,

If ORFS fails at any stage, is there any scripts to produce standalone test case like or_issue.py used in OpenLane flow : https://github.com/The-OpenROAD-Project/OpenLane/blob/master/docs/source/using_or_issue.md
Thanks in advance...
@maliberty  @vvbandeira

**Comment by vvbandeira** [comment, marked as answer]
You can create an issue for any Tcl step in `flow/scripts/`, e.g.:
```
make final_report_issue
make global_place_issue
```
The script at `flow/test/test_helper.sh` would create the issue automatically if the flow fails [and the env variable MAKE_ISSUE is set] ---- this is the methodology used by our CI.

(FYI, Matt is out this and next week)
